package.json

```diff
@@ -25,9 +25,10 @@
   "license": "Apache-2.0",
   "dependencies": {
     "moment": "^2.24.0",
+    "pino": "^6.11.0",
```

Comment:
Shall this be added later on when we will add the support ? I feel like this is not needed yet

Reply:
🤦